documentation/di-container/decorator.md

```diff
@@ -38,15 +38,15 @@ resolve1 !== resolve2;
 
 ## Custom DI Decorators
 
-We created some custom decorators to interact with the di-container. This decorators are needed for internal work. The complete framework use many shared classes and helpers on both sides. This decorators helps us to connected the server and client with the shared classes.
+We created some custom decorators to interact with the di-container. This decorators are needed for internal work. The entire framework uses many shared classes and helpers on both sides. This decorators helps us to connect the server and client with the shared classes.
```

Comment:
This should read `The decorator helps us to connect the server and client with the shared classes.`